### PR TITLE
Update <a> colors for dark and secondary sections

### LINKED ISSUE #1866: Low contrast hyperlink in the dark background sections
**Desired feature or idea.**
Improve hyperlink color when inside dark background sections. They currently also fail all WCAG levels.

image

### PR DESCRIPTION
**Pull request by raunaqgupta**
(Edit @chalin)

- Contributes to #1866

### Before

image

### After

image

### CODE REVIEW

**Comment by cartermp** [comment]
@chalin what do you think?

**Comment by chalin** [comment]
> @chalin what do you think?

Looking now.

File: assets/scss/_styles_project.scss [hunk 1]
```diff
@@ -118,17 +117,22 @@
...
-a:hover {
-  color: #3176d9;
-  text-decoration: underline;
+.td-box--secondary p > strong > a {
+  color: #18478b;
+
+  &:hover {
+    color: #18478b;
+  text-decoration: underline;  
+  }
```
Comment (chalin) [inline assets/scss/_styles_project.scss:135]: Nit: fix indentation

```suggestion
  &:hover {
    color: #18478b;
    text-decoration: underline;  
  }
```

**Review by chalin — APPROVED** [review]
Nice improvement, thanks. There are still some corner cases to address, and I'd like to tweak the styling a bit so it's less brittle. I'll merge this as is, and submit a followup PR; it'll be easier that way.

**Pull request merged**